src/NLog.WindowsIdentity/NLog.WindowsIdentity.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net45;net35;netstandard1.5;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(TargetFrameworks)' == '' ">net46;net45;net35;netstandard1.5;netstandard2.0;net6.0-Windows</TargetFrameworks>
```

Comment:
believe all should be lowercase, so `net6.0-Windows` -> `net6.0-windows`

Reply:
it's not case sensitive but i have switched it.

Reply:
Thanks then my OCD can relax